### DISCUSSION TITLE
Feature request: Explorer win7-style "status bar"

### DISCUSSION THREAD

**Discussion by eccccco** [valinet/ExplorerPatcher#3050] [Ideas]
Feature request/suggestion: Windows 7 style explorer with the big "status bar" where additional information about the selected file are visualized. I really miss the small preview thumbnail to the bottom left like in this example: https://www.sevenforums.com/tutorials/61523-status-bar-turn-off.html

In Win11 it's called "Detail Area" (or similiar) but it's placed to the right of the window and is kinda big.
There was a tool called OldNewExplorer that added this feature. But it doesn't work on Win11 any more.

**Comment by Gen-Jerod** [comment]
If only you can add this option.
